public/app/features/alerting/unified/components/rules/RulesFilter.tsx

```diff
@@ -98,23 +98,29 @@ const RulesFilter = () => {
           onChange={handleDataSourceChange}
           onClear={clearDataSource}
         />
-      </div>
+      </Field>
       <div className={cx(styles.flexRow, styles.spaceBetween)}>
         <div className={styles.flexRow}>
-          <div className={styles.rowChild}>
-            <Label>
-              <Tooltip
-                content={
-                  <div>
-                    Filter rules and alerts using label querying, ex:
-                    <pre>{`{severity="critical", instance=~"cluster-us-.+"}`}</pre>
-                  </div>
-                }
-              >
-                <Icon name="info-circle" className={styles.tooltip} />
-              </Tooltip>
-              Search by label
-            </Label>
+          <Field
+            className={styles.rowChild}
+            label={
+              <Label>
+                <Stack gap={0.5}>
+                  <span>Search by label</span>
+                  <Tooltip
+                    content={
+                      <div>
+                        Filter rules and alerts using label querying, ex:
+                        <code>{`{severity="critical", instance=~"cluster-us-.+"}`}</code>
+                      </div>
+                    }
+                  >
+                    <Icon name="info-circle" size="sm" />
+                  </Tooltip>
```

Comment:
think we should probably add a tooltip option to Field & Label component that renderings this icon and tooltip, feels like a common pattern that it would be good to share